### PR TITLE
Add ability to add stripe payment details to order admin.

### PR DESCRIPTION
**Pull request by umgelurgel**

### CODE REVIEW

File: conference/migrations/0014_auto_20190610_1352.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
```
Comment (artcz) [inline conference/migrations/0014_auto_20190610_1352.py]: It would be great to name the migration.

File: conference/models.py [hunk 2]
```diff
@@ -1595,7 +1595,7 @@ class StripePayment(models.Model):
         ('FAILED', 'Failed'),
     )
 
-    uuid = models.CharField(max_length=40, unique=True)
+    uuid = models.CharField(max_length=40, unique=True, default=uuid.uuid4)
```
Comment (artcz) [inline conference/models.py:1598]: Another possible option here would be to use short uuid, I think we already use it for Talks for example in the same module; (unless we already used uuid4 somewhere else in payments, then this default obviously makes more sense). WDYT?

**Review by artcz — APPROVED** [review]
Approved w/ comment about migration name.

File: conference/migrations/0014_auto_20190610_1352.py (hunk 1 above)
Comment (umgelurgel) [reply conference/migrations/0014_auto_20190610_1352.py]: Completely forgot - thanks for catching this :)

File: conference/models.py (hunk 2 above)
Comment (umgelurgel) [reply conference/models.py:1598]: We've been using `uuid4` for this until now - https://github.com/EuroPython/epcon/blob/ep2019/conference/cart.py#L170. I'd keep it as-is

**Pull request merged**